lib/linter-csslint.coffee

```diff
@@ -9,7 +9,7 @@ class LinterCsslint extends Linter
 
   # A string, list, tuple or callable that returns a string, list or tuple,
   # containing the command line (with arguments) used to lint.
-  cmd: 'csslint --format=compact'
+  cmd: ['csslint --format=compact']
```

Comment:
I don't think this initial value will be use anymore, will it? The call to `observe` below will initialize it.